### PR TITLE
Map logical structure from logical structmap.

### LINKED ISSUE #547: Logical Structure not rendering in UV
Not sure if it's not coming over, or what, but this item in dev:

https://hydra-dev.princeton.edu/concern/scanned_resources/sf2685136

has what I would want to be the page/canvas labels showing up in the UV's "Index". If you look in the Contents tab of the PUDL's viewer, it has two levels of structure:

http://pudl.princeton.edu/viewer.php?obj=tx31qh767#page/1/mode/1up

This is the relevant METS:

https://drive.google.com/open?id=0B4Wo5hgOEFY3Ri11eUpUYzgxRTQ

### PR DESCRIPTION
**Pull request by tpendragon**
Closes #547.

### CODE REVIEW

**Comment by tpendragon** [comment]
@escowles I think this is right, but I'm not really set up locally to actually try ingesting these. Could you test this when you get a chance?

**Comment by tpendragon** [comment]
Ingested item: https://hydra-dev.princeton.edu/concern/scanned_resources/st74cq51f#?c=0&m=0&s=0&cv=0

File: spec/models/mets_document_spec.rb [hunk 1]
```diff
@@ -4,28 +4,26 @@
   let(:mets_file) { Rails.root.join("spec", "fixtures", "pudl0001-4612596.mets") }
   let(:mets_file_rtl) { Rails.root.join("spec", "fixtures", "pudl0032-ns73.mets") }
   let(:mets_file_multi) { Rails.root.join("spec", "fixtures", "pudl0001-4609321-s42.mets") }
+  let(:mets_file_structure) { Rails.root.join("spec", "fixtures", "pudl0032-ns73.mets") }
```
Comment (escowles) [inline spec/models/mets_document_spec.rb]: Do we need to load pudl0032-ns73.mets twice?  Can't we just use the existing `mets_file_rtl`?

**Comment by escowles** [comment]
This looks good to me -- just one little comment.

**Comment by eliotjordan** [comment]
👍

**Pull request merged**